gcp/app-engine/README.md

```diff
@@ -29,6 +29,28 @@ e.g.
 curl http://127.0.0.1:8080/api/v1/tournaments
 ```
 
+To test slack without actually integration with slack, first run the app engine with slack integration disabled:
```

Comment:
FYI if you want to test locally without Slack integration, or with Slack integration that queries your locally-running app engine